### ISSUE TITLE
show desktop icon option

### ISSUE THREAD

**Issue by giffarus** [iabem97/komorebi#12]
when the option show desktop icon is unclick it's coming again... not that important but it's here. Therefore this option is not showing in the komorebi.prop

**Comment by iabem97** [comment]
Yes, the option was added but not functional yet since the desktop icons are not fully functional yet.
I've been really busy lately but I will be updating Komorebi this weekend!

**Comment by RobinEd** [comment]
Hi, I can see .txt desktop icons, but not .jpg and files with no extensions. Really great software, I really hope you find the time to keep updating this!
